src/angularjs/src/app/places/compare/compare.controller.js

```diff
@@ -28,9 +30,23 @@
                 return Places.getPlace(uuid);
             });
 
+            // first request score metadata
+            promises.unshift(ScoreMetadata.query().$promise);
```

Comment:
We should probably be caching most resources by default, and only override for requests expecting to pick up a change.